### ISSUE TITLE
[FEATURE] Export magic variables to env in subshell execution

### ISSUE THREAD

**Issue by MartyMcFlyInTheSky** [elkowar/eww#1165]
### Description of the requested feature

When executing a script from yuck, e.g.
```

(button :active {loading == false}
                    :class "spinner"
                    :onclick "python scripts/load_networks.py"
                    "${ spinner_symbol }")
```

populate its environment with the [magic variables](https://elkowar.github.io/eww/magic-vars.html) beforehand so the script can make use of them. E.g. in my python script I want to do the following:

```
eww_cmd: str = os.getenv('EWW_CMD')

...

    proc = await asyncio.create_subprocess_shell(f'{eww_cmd} update {vars}',
        stdout=asyncio.subprocess.PIPE,
        stderr=asyncio.subprocess.PIPE)
```

### Proposed configuration syntax

internal

**Comment by ovalkonia** [comment]
Why not just pass all the needed data as command line arguments?

**Comment by MartyMcFlyInTheSky** [comment]
I mean sure, but also why not have them as environmental variables? It would decrease the verbosity and make it more clean imho. As a user I was also quite surprised that this was not a thing, which goes against the principle of least astonishment.

**Comment by ovalkonia** [comment]
Fair enough. As it turns out, there's not way to access variables in `defvar`, `defpoll` and `deflisten`, so there is no way to pass them now. If we were able to do that, we could just pass the needed variables as command line arguments or just set the environmental variables ourselves. Having all magic variables set by default at all times seems kind of excessive, you realistically need one or two. Maybe the access to variables in `defvar` can be made possible

**Comment by MartyMcFlyInTheSky** [comment]
Yes that should be supported at least. Even though I think that exporting 10 variables is not all _that_ excessive, one could export just a subset. I mean for the other variables it might not even make sense to be exported, but `EWW_CMD` has a real use case.
